example/logo_7_8.py

```diff
@@ -1,26 +1,26 @@
 import logging
 import snap7
 
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
 # for setup the Logo connection please follow this link
 # http://snap7.sourceforge.net/logo.html
 
-
-logging.basicConfig(level=logging.INFO)
+logo_ip = ""    #  IP Adress of LOGO needed
 
 # Siemens LOGO devices Logo 8 is the default
 Logo_7 = True
-
-logger = logging.getLogger(__name__)
-
 plc = snap7.logo.Logo()
-plc.connect("192.168.0.41",0x1000,0x2000)
+plc.connect(logo_ip, 0x1000, 0x2000)
 
 if plc.get_connected():
     logger.info("connected")
     
     # read I1 from logo
-    vm_address = ("V923.0" if Logo_7==True else "V1024.0")
-    print ("I1: " + str(plc.read(vm_address)))
+    vm_address = ("V923.0" if Logo_7 is True else "V1024.0")
```

Comment:
this should just be == be in this case it probably doesn't matter much since it is a bool.

Reply:
Because it is a bool value, this would be the better style. In the logical way it wouldn't matter.

Reply:
as far is I know it works, this is not a better style. With `is` you check if the identity is equal, not the values. We are interested in the values here. It works with bools, but that doesn't mean it a better idea :)

Reply:
Alright, I will switch back '=='